components/common/src/command/package/install.rs

```diff
@@ -702,11 +704,38 @@ impl<'a> InstallTask<'a> {
     where
         T: UIWriter,
     {
-        artifact.unpack(Some(self.fs_root_path))?;
-        ui.status(Status::Installed, artifact.ident()?)?;
+        let ident = &artifact.ident()?;
+        let real_install_path = &pkg_install_path(ident, Some(self.fs_root_path));
+        let real_install_base = Path::new(real_install_path)
+            .parent()
+            .ok_or_else(|| Error::PackageUnpackFailed("Could not determine parent directory for package unpack location".to_owned()))?;
```

Comment:
This can be simplified a bit with `ok_or`:
```rust
        let real_install_base = Path::new(real_install_path).parent().ok_or(
            Error::PackageUnpackFailed(
                "Could not determine parent directory for package unpack location".to_owned(),
            ),
        )?;
```